mappings/net/minecraft/advancement/PlayerAdvancementTracker.mapping

```diff
@@ -5,6 +5,7 @@ CLASS net/minecraft/unmapped/C_hyeqtgvx net/minecraft/advancement/PlayerAdvancem
 	FIELD f_heqienju playerManager Lnet/minecraft/unmapped/C_digmgtxw;
 	FIELD f_ibwtrycy dataFixer Lcom/mojang/datafixers/DataFixer;
 	FIELD f_irznbzby owner Lnet/minecraft/unmapped/C_mxrobsgg;
+	FIELD f_jehxxcjj advancementProgresses Ljava/util/Map;
```

Comment:
oh my god this is a real word

Reply:
me neither, but i did check wiktionary; it says "[yep! it's plural of `progress`](https://en.wiktionary.org/wiki/progresses)"